### ISSUE TITLE
Formatter: pause not supported

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#3]
```
When exporting scripts from Selenium IDE, the formatters do not convert the 
pause command into a usable method in VBA/VBS.

Attached file was exported from Selenium IDE using the formatter. Running the 
script produces a Windows Script Host error:

"
Object doesn't support this property or method: 'pause'
"


```

Original issue reported on code.google.com by `matthewkastor@gmail.com` on 6 May 2012 at 4:29

Attachments:
- [pause.vbs](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-3/comment-0/pause.vbs)

**Comment by GoogleCodeExporter** [comment]
```
Edited the formatters to change pause commands in Selenium IDE to wait commands 
in VBA and VBS. Files Attached.
```

Original comment by `matthewkastor@gmail.com` on 6 May 2012 at 7:06

Attachments:
- [myVba Formatter.js](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-3/comment-1/myVba Formatter.js)
- [myVbs Formatter.js](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-3/comment-1/myVbs Formatter.js)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.5.0
```

Original comment by `florentbr` on 7 May 2012 at 8:08
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Awesome! Thank you! 
```

Original comment by `matthewkastor@gmail.com` on 7 May 2012 at 12:21
